### PR TITLE
LG notebook - Repair broken import and add note about dependency

### PR DESCRIPTION
**Pull request by tryrobbo**
Resolve an import issue and add a note explaining the dependency of the Evaluation section of the notebook on the llama package

### CODE REVIEW

File: recipes/responsible_ai/llama_guard/llama_guard_customization_via_prompting_and_fine_tuning.ipynb [hunk 1]
```diff
@@ -473,7 +473,9 @@
     "\n",
     "\n",
     "## Evaluation\n",
-    "The code below shows a workflow for evaluating the model using Toxic Chat. ToxicChat is provided as an example dataset. It is recommended that an dataset chosen specifically for the application be used to evaluate fine-tuning success. ToxicChat can be used to evaluate any degredation in standard category performance caused by the fine-tuning. \n"
+    "The code below shows a workflow for evaluating the model using Toxic Chat. ToxicChat is provided as an example dataset. It is recommended that an dataset chosen specifically for the application be used to evaluate fine-tuning success. ToxicChat can be used to evaluate any degredation in standard category performance caused by the fine-tuning. \n",
+    "\n",
+    "Note: This code relies on the llama package. To install if this is not yet installed: ```git clone https://github.com/meta-llama/llama/;cd llama;pip install -e .```\n"
```
Comment (mreso) [inline recipes/responsible_ai/llama_guard/llama_guard_customization_via_prompting_and_fine_tuning.ipynb:478]: I think we're not expecting people to alter the llama code, right? So why not just install through without cloning and install editable:
```suggestion
    "Note: This code relies on the llama package. To install if this is not yet installed: ```pip install  git+https://github.com/meta-llama/llama/ .```\n"
```

**Review by mreso — APPROVED** [review]
LGTM, just a nit regarding installation

File: recipes/responsible_ai/llama_guard/llama_guard_customization_via_prompting_and_fine_tuning.ipynb (hunk 1 above)
Comment (tryrobbo) [reply recipes/responsible_ai/llama_guard/llama_guard_customization_via_prompting_and_fine_tuning.ipynb:478]: I did not know that this was possible. Thanks!

**Review by tryrobbo — COMMENTED** [review]
Rolling this change into another PR

**Pull request closed without merging**